pom.xml

```diff
@@ -91,7 +92,7 @@
     <dependency.snakeyaml.version>2.2</dependency.snakeyaml.version>
     <dependency.spring.version>6.1.8</dependency.spring.version>
     <dependency.testcontainers.version>1.19.8</dependency.testcontainers.version>
-    <dependency.zeebe.version>8.6.6</dependency.zeebe.version>
+    <dependency.zeebe.version>8.6.7-rc1</dependency.zeebe.version>
```

Comment:
❓ Why this version ?

Reply:
I needed a release with https://github.com/camunda/camunda/commit/7c9485a27be1dfe3636a572465efedaaa743ef45 will be replaced with the official 8.6.7 patch once available next week.